src/js/components/Feedback/stories/FeedbackInline.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
For the inline version of the Feedback what should happen when you click 'Cancel'? Is the Feedback component no longer displayed?

Reply:
oh right good point maybe only show cancel if `modal`

Reply:
I still need to clarify the inline flow with designer but will bring up the `cancel`